packages/phrases/src/locales/de/translation/admin-console/upsell/paywall.ts

```diff
@@ -3,7 +3,6 @@ const paywall = {
     '{{count, number}} Anwendung von <planName/> erreicht. Plan upgraden, um den Bedürfnissen Ihres Teams gerecht zu werden. Für Unterstützung können Sie uns gerne <a>kontaktieren</a>.',
   applications_other:
     '{{count, number}} Anwendungen von <planName/> erreicht. Plan upgraden, um den Bedürfnissen Ihres Teams gerecht zu werden. Für Unterstützung können Sie uns gerne <a>kontaktieren</a>.',
-  /** UNTRANSLATED */
```

Comment:
tag removed but not translated in this file

Reply:
you are the man